sale_pricelist_auto_update/models/res_partner.py

```diff
@@ -24,6 +24,11 @@ class ResPartner(models.Model):
         readonly=True,
         string="Company Currency",
     )
+    fix_pricelist = fields.Boolean(
+        store=True,
+        default=False,
+        string="Fixed Pricelist",
```

Comment:
Not specific to this field, but would be good to have a help text to avoid confusion.

```
        help="If selected, the partner will be excluded from pricelist auto-update."
```